src/htrun/__init__.py

```diff
@@ -360,7 +360,7 @@ def init_host_test_cli_params():
     )
 
     parser.description = (
-        """Flash, reset and perform host supervised tests on mbed platforms"""
+        """Flash, reset and perform host supervised tests on Greentea supported hardware platforms"""
```

Comment:
What are "Greentea supported hardware platforms"?

Reply:
l thought to use the list from `https://github.com/ARMmbed/mbed-os-tools/blob/master/src/mbed_os_tools/detect/platform_database.py#L43` to document in htrun `Readme` what is your thoughts

Reply:
This `detect` module relies on "Mbed specific" information stored on the device. There's an MBED.HTM file which it pulls the product code from, to match with its internal database. Maybe we should just say "Mbed Enabled platforms" for now, unless there's a plan to make `detect` work with devices that don't follow the Mbed Enabled program?